pokedex-with-bit/app.py

```diff
@@ -25,18 +25,40 @@ def config():
 
 
 def index(num_docs: int):
-    f = Flow.load_config('flows/index.yml')
-    num_docs = min(num_docs, len(glob(os.path.join(os.getcwd(), IMAGE_SRC), recursive=True)))
+    num_docs = min(num_docs, len(glob(os.path.join(os.getcwd(), IMAGE_SRC),
+                                      recursive=True)))
+
+    f = Flow(workspace="workspace")\
```

Comment:
Could you change this to `flow`? let's try to avoid one letter var names

Reply:
Done: https://github.com/jina-ai/examples/pull/626/commits/f7e0191f8792cfd23956c6eaddcee9c834fcbd6f